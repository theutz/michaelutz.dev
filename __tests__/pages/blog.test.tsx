import { render, screen } from '@testing-library/react'
import Blog from 'pages/blog'

describe('<Blog />', () => {
  it('renders', () => {
    // Arrange

    // Act
    const { asFragment } = render(<Blog />)

    // Assert
    expect(asFragment()).toMatchSnapshot()
  })
})
