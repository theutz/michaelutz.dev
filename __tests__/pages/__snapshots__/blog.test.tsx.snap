// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Blog /> renders 1`] = `
<DocumentFragment>
  <div
    data-cy="page-blog"
  />
</DocumentFragment>
`;
